index.js

```diff
@@ -319,12 +319,13 @@ module.exports = function(mongoose) {
   var sendConfirmationEmail = function(email, callback) {
     var mailOptions = JSON.parse(JSON.stringify(options.confirmMailOptions));
     mailOptions.to = email;
-
-    if (!callback) {
-      callback = options.confirmSendMailCallback;
-    }
-    transporter.sendMail(mailOptions, callback);
-  };
+    if(shouldSendConfirmation){
```

Comment:
Space after `if` and before `{`. Also, change the tabs to 4 spaces in this block.